src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/resource/VideoLinkBookingControllerTest.kt

```diff
@@ -69,29 +71,46 @@ import java.util.Optional
 @ActiveProfiles("test")
 @WebAppConfiguration
 @WebMvcTest(controllers = [VideoLinkBookingController::class])
-@ContextConfiguration(classes = [VideoLinkBookingController::class])
+@ContextConfiguration(classes = [VideoLinkBookingController::class, VideoLinkBookingControllerTest.TestConfig::class])
 @WithMockUser(roles = ["BOOK_A_VIDEO_LINK_ADMIN"])
 class VideoLinkBookingControllerTest {
 
-  @MockBean
+  @TestConfiguration
+  class TestConfig {
+    @Bean
+    fun videoBookingRepository() = mock<VideoBookingRepository>()
+
+    @Bean
+    fun referenceCodeRepository() = mock<ReferenceCodeRepository>()
+
+    @Bean
+    fun videoAppointmentRepository() = mock<VideoAppointmentRepository>()
+
+    @Bean
+    fun locationsInsidePrisonClient() = mock<LocationsInsidePrisonClient>()
+
+    @Bean
+    fun videoLinkBookingsService() = VideoLinkBookingsService(
+      videoBookingRepository(),
+      referenceCodeRepository(),
+      videoAppointmentRepository(),
+      locationsInsidePrisonClient(),
+    )
+  }
+
+  @MockitoBean
   private lateinit var bookingFacade: BookingFacade
 
-  @MockBean
+  @MockitoBean
   private lateinit var requestBookingService: RequestBookingService
 
-  @SpyBean
```

Comment:
So here you are supposed to use MockitoSpyBean.  However there is a known issue with this annotation so I have added a test config to allow me to override the service directly instead.